assets/src/edit-story/components/panels/design/layer/layer.js

```diff
@@ -109,7 +103,6 @@ function Layer({ layer }) {
     currentPage: state.state.currentPage,
   }));
   const isBackground = currentPage.elements[0].id === layer.id;
-  const showPreview = !isBackground || layer.type !== 'shape';
 
   return (
     <LayerButton
```

Comment:
just chatted with Sam about the button states. She said these should follow the same rules as what we use for rectangle buttons.

I'll update this!